### DISCUSSION TITLE
extra large (150pt+) bold alpha numeric fonts?

### DISCUSSION THREAD

**Discussion by weeblebiker** [olikraus/u8g2#2410] [Q&A]
How can I get larger fonts than what is listed? the largest I found is u8g2_font_logisoso92_tn
With the advance in Oled displays with smaller pixels, most of the u8g2 library is too small to be useful and the larger fonts are limited.
I read mention of something called pixel doubling in u8x8, is something like that available?

**Comment by olikraus** [comment]
I think I had added X2 scaling once: https://github.com/olikraus/u8g2/issues/1919

**Reply by olikraus** [reply]
In fact the X2 option is mentioned in the docs: https://github.com/olikraus/u8g2/wiki/u8g2reference#drawstr

**Reply by weeblebiker** [reply]
thank you for the response, 
I have a color OLED display and am using u8g2 through Arduino_GFX
will this work with GFX also? how so, or is there another option?

**Reply by olikraus** [reply]
I think this was never ported back to the U8g2 for Adafruit GFX if this is the question...
